blocks/stat/stat.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * decorates the stat
+ * @param {Element} block The stat block element
+ */
+export default async function decorate(block) {
+  if (window.screen.width >= 768 && window.screen.width < 1200) {
+    const statParent = block.firstElementChild;
+    const childLenth = statParent.children.length;
+    if (childLenth === 3) {
+      [...statParent.children].forEach((ele) => {
+        ele.style.maxWidth = 'calc((100%/3) - 16px)';
+      });
+    } else if (childLenth === 2) {
+      [...statParent.children].forEach((ele) => {
+        ele.style.maxWidth = 'calc(50% - 12px)';
+      });
+    } else {
+      statParent.classList.add('no-effect');
+    }
+  }
+}
```

Comment:
We required to do so because width of the stats column section changes with the content.
In design also there is an issue where "%" sign is getting cut (screenshot added below), when we are not providing max-width then the column width gets automatically adjusted and it's not matching the design. Sometime the width becomes 228px and to accommodate this extra width, the width of 2nd column is getting reduced from 216px to ~194px. Hence, we required to add this CSS through JS as I was not able to figure out when there are 2 columns and when 3 columns are added to this section.

if you have any solution where we can avoid JS, please let me know, I'll try to include that in the code.

<img width="1270" alt="Screenshot 2023-08-31 at 3 39 19 PM" src="https://github.com/hlxsites/merative2/assets/135624814/e10780ee-9b78-492d-83c0-375185db6c08">

Reply:
We can give a try by reducing the letter spacing by -2%. i updated that for review in design. Hope this works.

@proeung @sahmad-merative @anabarcelona